src/opera/template/node.py

```diff
@@ -197,10 +197,10 @@ def get_artifact(self, params):
             raise DataError("Cannot find artifact '{}'.".format(prop))
 
     def concat(self, params):
-        return self.topology.concat(params)
+        return self.topology.concat(params, self.name)
```

Comment:
What is the point of providing name an then searching by name later? Why not provide node reference directly like
```suggestion
        return self.topology.concat(params, self)
```

Reply:
There's no point at all, so this has been optimized.